search/search-client-query-transformer/src/main/java/io/camunda/search/clients/query/SearchQueryBuilders.java

```diff
@@ -453,7 +454,7 @@ public static <C extends List<Operation<OffsetDateTime>>> List<SearchQuery> date
     }
   }
 
-  public static <C extends List<Operation<Object>>> List<SearchQuery> variableOperations(
+  public static <C extends List<UntypedOperation>> List<SearchQuery> variableOperations(
```

Comment:
💭 This will probably reverted back to just string instead of object. At least for the `variables/search` this is not needed yet.